docs/product-getting-started-guide/src/main/asciidoc/data-model-proc.adoc

```diff
@@ -12,19 +12,19 @@ The *Mortgage_Process* data model is comprised of four data objects:
 
 .Prerequisites
 
-Opened the Mortgage_Process sample project. For further information, see <<creating_business_project>>.
+Opened the Mortgage Process sample project. For further information, see <<creating_business_project>>.
 
 == Creating the Applicant data object
-For this tutorial, you will delete the pre-defined *Applicant* data object and then recreate it for practice. The three remaining objects are already defined and included in the *Mortgage_Process* data model.
+For this tutorial, you will delete the pre-defined *Applicant* data object and then recreate it for practice. The three remaining objects are already defined and included in the *Mortgage Process* data model.
```

Comment:
I thought we decided to include the existing "Data objects" and "Creating data objects" topics here instead. We discussed that, per Jozef's and my suggestion, and I shared to the adoc file names with you. You'll find them in any of my "Designing a decision service using..." docs. I'm not a fan of the deleting and re-creating thing, but it's up to you.

The majority of your procedures are missing the .Procedure heading. Look through all procedures and make sure each one has the .Procedure heading. Might be good to review the mod doc templates: https://mojo.redhat.com/docs/DOC-1099486#jive_content_id_Master_source_files_in_asciidoc_format